changelogs/fragments/mod_args.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Warn user if the reserved keyword _ansible_ used as a module parameter (https://github.com/ansible/ansible/issues/82514).
```

Comment:
```suggestion
  - Warn user if the reserved keyword ``_ansible_`` used as a module parameter (https://github.com/ansible/ansible/issues/82514).
```

Reply:
Although, I'd rephrase — the current description implies there was not warning, but there was a warning, it's just the wording that got extended.